syntax = "proto3";

package transaction;

message SumRequest {
	int64 user_id    = 1;
	int64 time_start = 2;
	int64 time_end   = 3;
}

message SumResponse {
	float total = 1;
}

service Sum {
	rpc sum_amount(SumRequest) returns (SumResponse) {}
}